Drop frame timecode support

Currently OTIO doesn't support drop-frame time signatures in any way.

Hi!
I've begun working on this and have a first version of the `from_timecode` rewrite done, but when I run the tests I get the fails listed below.
I understand why they fail, but the thing I don't understand is the behavior of the `from_frames` function. What is the purpose of the 600 time scale?
I realize I should add drop frame support in all aspects of the code, but I just want to understand the original code before I change anything.

```
======================================================================
FAIL: test_timecode_23976_fps (test_opentime.TestTime)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/data/dev/OpenTimelineIO/tests/test_opentime.py", line 184, in test_timecode_23976_fps
    self.assertEqual(t, t1)
AssertionError: otio.opentime.RationalTime(value=1025, rate=600) != otio.opentime.RationalTime(value=41, rate=23.976)

======================================================================
FAIL: test_timecode_2997fps (test_opentime.TestTime)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/data/dev/OpenTimelineIO/tests/test_opentime.py", line 210, in test_timecode_2997fps
    self.assertEqual(t, t1)
AssertionError: otio.opentime.RationalTime(value=940, rate=600) != otio.opentime.RationalTime(value=47, rate=29.97) 
```

--- a/docs/tutorials/otio-timeline-structure.md
+++ b/docs/tutorials/otio-timeline-structure.md
@@ -10,7 +10,7 @@ audio/video timeline.
 Rendering of the image tracks in a timeline is done in painter order. The layers
 in a stack are iterated from the bottom (the first entry in the stack) towards
 the top (the final entry in the stack). Images in a stack overlay lower images
-with no blending.
+using an alpha composite operation respecting any alpha in the source materials.
 
 Within a track, clips may overlap via a transition. In that case, the 
 contribution of track is the linear blend of the elements joined by the

--- a/docs/tutorials/otio-timeline-structure.md
+++ b/docs/tutorials/otio-timeline-structure.md
@@ -5,6 +5,26 @@ clips, gaps, and transitions. This document is meant to clarify how these
 objects nest within each other, and how they work together to represent an
 audio/video timeline.
 
+## Rendering
+
+Rendering of the image tracks in a timeline is done in painter order. The layers
+in a stack are iterated from the bottom (the first entry in the stack) towards
+the top (the final entry in the stack). Images in a stack overlay lower images
+with no blending.
+
+Within a track, clips may overlap via a transition. In that case, the 
+contribution of track is the linear blend of the elements joined by the
+transition.
+
+If there are effects on a clip, OpenTimelineIO does not say anything about the
+impact of the effect and deviation from the base behavior is application
+specific.
+
+Rendering of the audio tracks is additive. It is strongly advised, but not
+required, that the summed audio is summed as floating point, and that it is 
+processed through a compression filter in order to prevent clipping and 
+distortion. 
+
 ## Simple Cut List
 
 Let’s start with a simple cut list of a few clips. This is stored as a 

--- a/docs/tutorials/otio-timeline-structure.md
+++ b/docs/tutorials/otio-timeline-structure.md
@@ -11,7 +11,7 @@ Rendering of the image tracks in a timeline is done in painter order. The layers
 in a stack are iterated from the bottom (the first entry in the stack) towards
 the top (the final entry in the stack). Images in a stack overlay lower images
 using an alpha composite operation respecting any alpha in the source materials.
-
+All compositing is assumed to occur over a background of zero values in color components, and 100% values for alpha components.
 Within a track, clips may overlap via a transition. In that case, the 
 contribution of track is the linear blend of the elements joined by the
 transition.

